pkg/services/serviceaccounts/manager/service.go

```diff
@@ -289,3 +292,38 @@ func validAPIKeyID(apiKeyID int64) error {
 	}
 	return nil
 }
+
+// hideApiKeysTabIfNoAPIKeysPresent is used to remove the apikeys tab from the admin tabs
+// if there are no apikeys, this makes it so that we only issue service account tokens instead
+func hideApiKeysTabIfNoAPIKeysPresent(l *log.ConcreteLogger, store *sqlstore.SQLStore, kvStore kvstore.KVStore) {
+	var err error
+	ctx := context.Background()
+	sess := store.GetSqlxSession()
+	orgs := make([]*org.OrgDTO, 0)
+	err = sess.Select(ctx, &orgs, "SELECT id, name FROM org")
+	if err != nil {
+		l.Error(fmt.Sprintf("unable to query for org for hiding api keys: %e", err))
+		return
+	}
+	if len(orgs) != 1 {
+		l.Info("do not want to hide apikeys for more than 1 org")
+		return
+	}
+
+	orgID := orgs[0].ID
+	apikeys := make([]*apikey.APIKey, 0)
+	if orgID != -1 {
+		err = sess.Select(ctx, &apikeys, "SELECT * FROM api_key where org_id =?", orgID)
```

Comment:
Here you query both API keys and service account tokens I guess. I think it makes sense to put a comment about it. Also, `count` query might be more efficient here.

Reply:
implemented count